app/calculator.py

```diff
@@ -0,0 +1,16 @@
+import streamlit as st
+
+def calculate(choice, num_1: float, num_2:float):
+    if choice == "Addition":
+         result = num_1 + num_2
```

Comment:
This is good, however if you read the repo README, you will see that I have include 

> To implement modularization, single responsibility (SOLID) and low decoupling, high cohesion design.

Which means that, the operation should me modularize (within a function). For example 

```
# in utils.py
def addition(value_1, value_2):
    """
    Function will return the sum of 2 value, value_1 and value_2. 
    Params:
     - value_1:int - first value
     - value_2: int - second calue
     
    Return:
    - sum of value_1 and value_2

    """
    return value_1 + value_2
```
```
# in calculator.py
from utils.import addition
def calculate(choice, num_1: float, num_2:float):
    if choice == "Addition":
         result = addition(value_2, value_2)
```

Which also applied the Single Responsibility principle and more abstraction.